src/lib.rs

```diff
@@ -304,19 +282,22 @@ fn parse_attr(attr: &syn::Attribute, mode: GenMode) -> Option<Meta> {
                 skip
             } else {
                 abort!(
-                    last.or_else(|| collected.pop()).unwrap().path().span(),
+                    last.or_else(|| collected.into_iter().next())
```

Comment:
I think pop makes more sense here rather than constructing an intermediate iterator just to return one item.